### DISCUSSION TITLE
How were the structure overlays in the docs made?

### DISCUSSION THREAD

**Discussion by archianese** [grimme-lab/xtb#732] [Q&A]
I'm not sure if this is an ok place for this question, but I'm wondering how the structure overlay graphics in the CREST documentation were made.

A really nice example is this one:
![image](https://user-images.githubusercontent.com/119081251/204065138-1a36c81e-c4a0-462a-ad82-b72ff9b97fb4.png)
 
which is on this page:
https://xtb-docs.readthedocs.io/en/latest/crestxmpl.html#conformers-ts

**Comment by codecollider** [comment]
You can do such overlays with e.g. Chemcraft.

**Comment by icamps** [comment]
You can do also with Jmol, Chimera and MAESTRO.
